plugin/src/main/java/com/denizenscript/denizen/scripts/commands/entity/FakeInternalDataCommand.java

```diff
@@ -0,0 +1,120 @@
+package com.denizenscript.denizen.scripts.commands.entity;
+
+import com.denizenscript.denizen.nms.NMSHandler;
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizen.objects.PlayerTag;
+import com.denizenscript.denizen.utilities.Utilities;
+import com.denizenscript.denizencore.DenizenCore;
+import com.denizenscript.denizencore.exceptions.InvalidArgumentsRuntimeException;
+import com.denizenscript.denizencore.objects.core.DurationTag;
+import com.denizenscript.denizencore.objects.core.MapTag;
+import com.denizenscript.denizencore.scripts.ScriptEntry;
+import com.denizenscript.denizencore.scripts.commands.AbstractCommand;
+import com.denizenscript.denizencore.scripts.commands.generator.*;
+import com.denizenscript.denizencore.utilities.debugging.Debug;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.Player;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.locks.LockSupport;
+
+public class FakeInternalDataCommand extends AbstractCommand {
+
+    // <--[language]
+    // @name Internal Entity Data
+    // @group Minecraft Logic
+    // @description
+    // Each entity in Minecraft has a set of data values that get sent to the client, with each data value being a number id -> value pair.
+    // Denizen allows direct control over that data, as it can be useful for things like setting values that would usually be blocked.
+    // Because this is such a direct control that's meant to impose less restrictions, there's no limitations/verification on the values being set other than basic type checking.
+    // For all possible internal entity data values and their respective ids, see <@link url https://github.com/DenizenScript/Denizen/blob/dev/v1_20/src/main/java/com/denizenscript/denizen/nms/v1_20/helpers/EntityDataNameMapper.java#L50>.
```

Comment:
This doc should note somewhere that these values are liable to change between minecraft versions